### DISCUSSION TITLE
I've received an invitation to GitHub Copilot, yet it seems to say I need to sign up for the waitlist

### DISCUSSION THREAD

**Discussion by munchkindev** [community/community#14229] [Copilot Conversations] [answered]
Hello,

As the title already says, I've been invited to test drive GitHub Copilot and when I enter the personal access token from GitHub with rights to `read:user` it seems to say that I need to sign up for the waitlist, even though I used the account that's bound to my E-mail address.

Help is appreciated.

**Comment by davecheney** [comment]
@munchkindev which editor are you using?

**Comment by munchkindev** [comment, marked as answer]
@davecheney  VS Code, VSCodium to be exact

**Reply by davecheney** [reply]
Sorry to tell you that only VScode works at the moment. Authentication does not currently work with the source rebuilt versions.

**Reply by munchkindev** [reply]
Will it be supported in the future?

**Reply by davecheney** [reply]
I know that the developers who work on the extension are aware of the problem so I would imagine it’s on their roadmap, but I can’t give you any details about when they might get too it.

**Reply by munchkindev** [reply]
I see, thanks. I know this is out of the scope, but can I have VSCode and VSCodium on the same machine running at the same time?

**Reply by davecheney** [reply]
Yes
